[2024-09-01 19:13:11,331] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: exchange_rates_dag.run_main_script scheduled__2024-05-28T00:00:00+00:00 [queued]>
[2024-09-01 19:13:11,413] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: exchange_rates_dag.run_main_script scheduled__2024-05-28T00:00:00+00:00 [queued]>
[2024-09-01 19:13:11,420] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-09-01 19:13:11,422] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2024-09-01 19:13:11,431] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-09-01 19:13:11,568] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): run_main_script> on 2024-05-28 00:00:00+00:00
[2024-09-01 19:13:11,600] {standard_task_runner.py:52} INFO - Started process 1261 to run task
[2024-09-01 19:13:11,637] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'exchange_rates_dag', 'run_main_script', 'scheduled__2024-05-28T00:00:00+00:00', '--job-id', '153', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates_dag.py', '--cfg-path', '/tmp/tmp3qobk372', '--error-file', '/tmp/tmpqkg8mbvy']
[2024-09-01 19:13:11,650] {standard_task_runner.py:80} INFO - Job 153: Subtask run_main_script
[2024-09-01 19:13:12,272] {task_command.py:371} INFO - Running <TaskInstance: exchange_rates_dag.run_main_script scheduled__2024-05-28T00:00:00+00:00 [running]> on host 6de2165e2444
[2024-09-01 19:13:12,953] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rates_dag
AIRFLOW_CTX_TASK_ID=run_main_script
AIRFLOW_CTX_EXECUTION_DATE=2024-05-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-28T00:00:00+00:00
[2024-09-01 19:13:32,227] {python.py:173} INFO - Done. Returned value was: None
[2024-09-01 19:13:32,748] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=exchange_rates_dag, task_id=run_main_script, execution_date=20240528T000000, start_date=20240901T191311, end_date=20240901T191332
[2024-09-01 19:13:33,081] {local_task_job.py:221} WARNING - State of this instance has been externally set to success. Terminating instance.
[2024-09-01 19:13:33,167] {process_utils.py:129} INFO - Sending Signals.SIGTERM to group 1261. PIDs of all processes in the group: [1261]
[2024-09-01 19:13:33,177] {process_utils.py:80} INFO - Sending the signal Signals.SIGTERM to group 1261
[2024-09-01 19:13:33,183] {process_utils.py:75} INFO - Process psutil.Process(pid=1261, status='terminated', exitcode=0, started='19:13:10') (1261) terminated with exit code 0
